dashboard/src/components/ConfirmDialog/index.tsx

```diff
@@ -33,22 +31,21 @@ class ConfirmDialog extends React.Component<IConfirmDialogProps, IConfirmDialogS
               transform: "translate(-50%, -50%)",
             },
           }}
+          ariaHideApp={false}
           isOpen={this.props.modalIsOpen}
           onRequestClose={this.closeModal}
           contentLabel="Modal"
         >
-          {this.state.error && (
-            <div className="padding-big margin-b-big bg-action">{this.state.error}</div>
-          )}
           {this.props.loading === true ? (
             <div> Loading ... </div>
           ) : (
             <div>
               <div> Are you sure you want to delete this? </div>
-              <button className="button" onClick={this.props.closeModal}>
+              <button id="cancel" className="button" onClick={this.props.closeModal}>
```

Comment:
A better option is to use React's `ref` - https://reactjs.org/docs/refs-and-the-dom.html https://airbnb.io/enzyme/docs/api/ReactWrapper/ref.html.